common/common.go

```diff
@@ -595,3 +599,142 @@ func GetFuncYamlVersion(oldFF map[string]interface{}) int {
 	}
 	return 1
 }
+
+// UserConfirmedMultiResourceDeleteion will prompt the user for confirmation to delete all the the resources
+func UserConfirmedMultiResourceDeleteion(aps []*modelsv2.App, fns []*modelsv2.Fn, trs []*modelsv2.Trigger) bool {
+
+	apsLen := len(aps)
+	fnsLen := len(fns)
+	trsLen := len(trs)
+
+	fmt.Println("You are about to delete the following resources:")
+	if apsLen > 0 {
+		fmt.Println("   Applications:", apsLen)
+	}
+	if fnsLen > 0 {
+		fmt.Println("   Functions:   ", fnsLen)
+	}
+	if trsLen > 0 {
+		fmt.Println("   Triggers:    ", trsLen)
+	}
+	fmt.Println("This operation cannot be undone")
+	fmt.Printf("Do you wish to proceed? Y/N: ")
+	reader := bufio.NewReader(os.Stdin)
+	input, _ := reader.ReadString('\n')
+	if strings.ToLower(input) == "y\n" {
```

Comment:
You might want to consider running this through `strings.TrimSpace` to get rid of any spaces and newlines (https://golang.org/pkg/strings/#TrimSpace)